packages/devtools_app/lib/src/screens/network/network_controller.dart

```diff
@@ -247,12 +218,13 @@ class NetworkController extends DisposableController
   Future<void> clear() async {
     await _networkService.clearData();
     _requests.value = NetworkRequests();
+    _currentNetworkRequests.clear();
     resetFilter();
-    _updateData();
+    _handleRequestListChanges();
     _updateSelection();
   }
 
-  void _updateData() {
+  void _handleRequestListChanges() {
```

Comment:
maybe rename to _filterAndRefreshSearchMatches() to make it clear what this does?